models/post-comment.model.js

```diff
@@ -0,0 +1,21 @@
+const db = require("../db/connection")
+
+exports.insertComment = (articleId, postComment) => {
+
+return db.query(
+
+`INSERT INTO comments
+(body,author, article_id) VALUES ($1, $2, $3)
+RETURNING *; `,
+[postComment.body ,postComment.username, articleId]
+
+).then((comment) => {
+
+    console.log(comment)
+
+return comment.rows[0]
```

Comment:
This seems to be fore ticket 7 which should ideally not be part of a PR for ticket 6. Try remember to keep tickets seperate.